modules/integration/tests-ui-integration/src/test/java/org/wso2/am/integration/ui/tests/APIMANAGER3026DescriptionFieldTestCase.java

```diff
@@ -0,0 +1,100 @@
+/*
+ * Copyright (c) 2015​, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ * 
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ * 
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.wso2.am.integration.ui.tests;
+
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.testng.Assert;
+import org.testng.annotations.AfterClass;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+import org.wso2.am.integration.test.utils.bean.APIRequest;
+import org.wso2.am.integration.test.utils.publisher.utils.APIPublisherRestClient;
+import org.wso2.carbon.automation.extensions.selenium.BrowserManager;
+
+import java.net.URL;
+
+/**
+ * The issue in APIMANAGER3026 is when api is created by rest call via publisher APIs
+ * and when we try to edit description of created API, the description was not found.
+ * This was fixed and will test by this test case.
+ */
+public class APIMANAGER3026DescriptionFieldTestCase extends AMIntegrationUiTestBase {
+
+	private WebDriver driver;
+	private APIPublisherRestClient apiPublisher;
+	private String userName = "admin";
```

Comment:
hard coding user names and passwords not recommended. you can read values though automation context